orderbook-subgraph/src/ExchangeMapping.ts

```diff
@@ -134,24 +92,8 @@ export function handleFill(event: OrderFilled): void {
     side,
   );
 
-  updateUserVolume(taker, size, collateralScaleDec, timestamp);
-  markAccountAsSeen(taker, timestamp);
-
-  updateUserVolume(maker, size, collateralScaleDec, timestamp);
-  markAccountAsSeen(maker, timestamp);
-
   updateTradesQuantity(orderBook, side);
 
-  // Update market data with most recent orderbook price
-  let marketData = MarketData.load(tokenId);
-  if (marketData != null) {
-    marketData.priceOrderbook = enriched.price;
-    marketData.save();
-  }
-
-  // Update market position
-  updateMarketPositionFromOrderFilled(maker, tokenId, side, event);
```

Comment:
we no longer update market position?

Reply:
Deprecating Market position entirely, it's no longer used anywhere